File: src/main/java/site/gachontable/presentation/shared/DateTimeFormatters.java
package site.gachontable.presentation.shared;

import java.time.format.DateTimeFormatter;
import java.util.Locale;

public class DateTimeFormatters {

    public static final DateTimeFormatter WITH_WEEKDAY = DateTimeFormatter.ofPattern(
            "M월 d일 (E) HH:mm", Locale.KOREAN);

}

File: src/main/java/site/gachontable/presentation/shared/DateTimeFormatters.kt
package site.gachontable.presentation.shared

import java.time.format.DateTimeFormatter
import java.util.*

object DateTimeFormatters {
    val WITH_WEEKDAY: DateTimeFormatter = DateTimeFormatter
        .ofPattern(
            "M월 d일 (E) HH:mm", Locale.KOREAN
        )
}
